test/module/irohad/torii/torii_queries_test.cpp

```diff
@@ -455,6 +456,68 @@ TEST_F(ToriiQueriesTest, FindSignatoriesHasRolePermissions) {
   ASSERT_EQ(response_pubkey, pubkey);
 }
 
+/**
```

Comment:
This `TEST` is almost same as `develop`. Different point is changing getAccountTransactions() using pager.